ll_insertions.py

```diff
@@ -0,0 +1,73 @@
+from node import Node
+
+class LinkedList:
+    """ class for new linked list """
+    def __init__(self, iter = []):
+        self.head = None
+        self._len = 0
+
+
+        for item in iter:
+            self.head = Node(item, self.head)
+            self.len += 1
+
+
+def __len__(self):
```

Comment:
All of these function definitions need to be indented. Currently they are not actually methods on the `LinkedList` class because they are not defined within the class.